### ISSUE TITLE
Error no such file ./node_modules/ngentest/src/templates/default.spec.ts.ejs

### ISSUE THREAD

**Issue by sujeethanv** [allenhwkim/ngentest#8]
Error: ENOENT: no such file or directory, open './node_modules/ngentest/src/templates/default.spec.ts.ejs'
    at Object.fs.openSync (fs.js:663:18)
    at Object.fs.readFileSync (fs.js:568:33)
    at Object.getEjsTemplate (./node_modules/ngentest/src/lib/util.js:26:13)
    at parseTypescript.then.tsParsed (./node_modules/ngentest/index.js:28:28)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
    at Function.Module.runMain (module.js:703:11)
    at startup (bootstrap_node.js:193:16)
    at bootstrap_node.js:617:3

Happens in file with 2 export functions as the content of the .ts file.

**Comment by allenhwkim** [comment]
Please provide a `.ts` file for this error case.

**Comment by lk77** [comment]
Hello,
i have the same error :
```
Error: ENOENT: no such file or directory, open '/usr/local/lib/node_modules/ngentest/src/templates/default.spec.ts.ejs'
    at Object.fs.openSync (fs.js:663:18)
    at Object.fs.readFileSync (fs.js:568:33)
    at Object.getEjsTemplate (/usr/local/lib/node_modules/ngentest/src/lib/util.js:26:13)
    at parseTypescript.then.tsParsed (/usr/local/lib/node_modules/ngentest/index.js:28:28)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
    at Function.Module.runMain (module.js:703:11)
    at startup (bootstrap_node.js:190:16)
    at bootstrap_node.js:662:3
```

**Comment by lk77** [comment]
@allenhwkim it's an ngModule

**Comment by allenhwkim** [comment]
This has been fixed and published as 0.3.7

**Comment by lk77** [comment]
@allenhwkim thanks !
